### DISCUSSION TITLE
Feedback: Can you consider incorporating Julia as well, as one of the available option, like Python?

### DISCUSSION THREAD

**Discussion by kizombaciao** [community/community#34899] [Codespaces]
Feedback: Can you consider incorporating Julia as well, as one of the available option, like Python?

**Comment by adnamalin** [comment]
👋  Julia is supported in VSCode (https://www.julia-vscode.org/, https://code.visualstudio.com/docs/languages/julia) so you could add it to your codespaces by [setting up a dev container configuration](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers). GitHub does provide a [default dev container configuration](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers#using-the-default-dev-container-configuration) that includes popular languages such as Python, Node, PHP, Java, Go, C++, Ruby, and .NET Core/C#.
